lang/csharp/common.props

```
@@ -45,6 +46,7 @@
 
   <ItemGroup>
     <None Include="$(MSBuildThisFileDirectory)\LICENSE" Pack="true" Visible="false" PackagePath=""/>
+    <None Include="$(MSBuildThisFileDirectory)/../../doc/src/resources/images/avro-logo.png" Pack="true" PackagePath="\"/>
```

Could you set `Visible="false"` in this, like with did with `LICENSE` above? That prevents this file from appearing in the project tree view in Visual Studio. I think `PackagePath` should be empty like in `LICENSE` as well. Did you get an error with it being blank?

```suggestion
    <None Include="$(MSBuildThisFileDirectory)/../../doc/src/resources/images/avro-logo.png" Pack="true" Visible="false" PackagePath=""/>
```